files/image_config/monit/memory_checker

```diff
@@ -96,6 +96,11 @@ def check_memory_usage(container_name, threshold_value):
         sys.exit(4)
 
 
+def is_service_active(service):
+    status = subprocess.run("systemctl is-active --quiet {}".format(service), shell=True, check=False)
```

Comment:
Can we use the command `systemctl is-active docker` to check whether the `docker daemon` is active or not? Then we can ues the existing function at line 31 `get_command_result(...)`.

Reply:
Can we also add comments for this function to explain why we need this function?

Reply:
I wanted to use get_command_result() but couldnt from the following reasons:
1. It returns only the command output. I do not need the command output, I need to test the command return code.
2. If command return code is not 0, it fails with error to log and sys.exit(1). In my case return code not 0 means that docker daemon is not running which is a valid flow.

I added function documentation.